lib/ruby_lsp/executor.rb

```diff
@@ -545,6 +546,19 @@ def detected_formatter
       end
     end
 
+    sig { returns(String) }
+    def detected_test_library
+      if direct_dependency?(/^minitest/)
+        "minitest"
+      elsif direct_dependency?(/^test-unit/)
+        "test-unit"
+      elsif direct_dependency?(/^rspec/)
```

Comment:
One could argue that the RSpec check should be specified only as part a RSpec extension, but I'm not too bothered.

Reply:
IMO we should detect it here so we don't print the "No test library detected" message for RSpec users.